### PR TITLE
Improving documentation for PAYG integration

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Detailed and informative comments on bug #1208984 have been added to the documentation.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 https://github.com/uyuni-project/uyuni-docs/pull/2236
- [x] Manager-4.2 https://github.com/uyuni-project/uyuni-docs/pull/2237

# Links

Fixes https://github.com/SUSE/spacewalk/issues/20709

### CODE REVIEW

**Comment by 0rnela** [comment]
@rjmateus -- as you will see, this is just skeleton text. I had used the content from the [bug 1208984 comment #12](https://bugzilla.suse.com/show_bug.cgi?id=1208984#c12) and tried to rearrange it to the best of my ability for now.

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (rjmateus) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:145]:
```suggestion
* Refreshing or changing the [litera;]``registercloudguest`` connection to the public cloud update infrastructure should not interfere with the credentials usage.
```

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (rjmateus) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:161]: This section about "== {payg} connector process" I think can be replaced by a warning or suggestion.
 
```suggestion
Any of the following actions/changes in the payg instance will lead to credentials stopping to work:
* Remove zypper credentials files
* Remove the imported certificates
* Remove cloud-specific entries from `/etc/hosts`
* Stop the service named "??"
```

**Review by rjmateus — COMMENTED** [review]
Hey Ornela,
I added some comments. I think we can have just a few alerts

**Comment by 0rnela** [comment]
@rjmateus -- I implemented the changes we talked about. Now I'm just waiting for the service name, and I can ask for approval from other reviewers too.

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (rjmateus) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:136]:
```suggestion
* Running [literal]```registercloudguest --clean`` will cause problems if no new cloud connection is registered with the cloud guest command.
```

**Review by rjmateus — COMMENTED** [review]
a few notes

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (0rnela) [reply modules/installation-and-upgrade/pages/connect-payg.adoc:134]: Done.

File: .changelog
Comment (keichwa) [inline .changelog:8]:
```suggestion
- Improved Pay-as-you-go documentation in the Install and Upgrade
  Guide (bsc#1208984)
```

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (keichwa) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:105]: Wondering whether we can delete the expression "up to date" here?

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (keichwa) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:135]:
```suggestion
The rest of the process is the same as for any non-public-cloud client and consists of synchronizing channels, automatic bootstrap script creation, activation key creation and starting the registration.
```

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (keichwa) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:146]: Why writing "remove" here?

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (keichwa) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:152]:
```suggestion
Checking the credentials::
```

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (keichwa) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:153]:
```suggestion
* If the script fails to collect the credentials, it should provide a proper error message in the logs and in the {webui}.
```

**Review by keichwa — CHANGES_REQUESTED** [review]

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/connect-payg.adoc:105]: "The latest status of the {payg} instance credentials collect is show in....

Comment (0rnela) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:105]:
```suggestion
The status of the {payg} instance credentials collect is shown in the column [literal]``Status`` or on the instance details page.
```

File: modules/installation-and-upgrade/pages/connect-payg.adoc
Comment (0rnela) [inline modules/installation-and-upgrade/pages/connect-payg.adoc:146]:
```suggestion
* removing cloud-specific entries from [path]``/etc/hosts``
```

**Review by keichwa — APPROVED** [review]
thanks!

**Pull request merged**
